auto_install.sh

```diff
@@ -79,6 +82,9 @@ while getopts "$OPTIONS" opt ; do
         g )
             GPIO_KEY_WORD_DETECTOR_FLAG="-g"
             ;;
+        k )
+            HID_KEY_WORD_DETECTOR_FLAG="-k"
```

Comment:
Have you considered using `G` for GPIO and `H` for HID?

Reply:
I updated the options